File: docs/day01.js
document.addEventListener("DOMContentLoaded", () => {
  const runButton = document.getElementById("runButton");
  const output = document.getElementById("output");

  runButton.addEventListener("click", () => {
    output.textContent = "Hi there and welcome to the Band Name Generator 🎸\n";
    
    setTimeout(() => {
      const city = prompt("Which city did you grow up in?");
      if (!city) return;

      output.textContent += `Which city did you grow up in?\n> ${city}\n`;

      setTimeout(() => {
        const pet = prompt("What is the name of your pet?");
        if (!pet) return;

        output.textContent += `What is the name of your pet?\n> ${pet}\n\n`;
        output.textContent += `Your band name could be: ${city} ${pet}`;
      }, 300);
    }, 300);
  });
});

File: days/day01/day01.py
# 🎸 Band Name Generator
# Day 01 of 100 Days of Code – Python Bootcamp

def generate_band_name():
    print("👋 Greetings, Stranger!")
    
    # Prompt the user for inputs
    city = input("🌆 Which city did you grow up in?\n").strip()
    pet = input("🐶 What is the name of your pet?\n").strip()
    
    # Generate and display the band name
    band_name = f"{city} {pet}"
    print(f"\n🎤 Your band name could be: {band_name}")

# Run the generator
generate_band_name()
